packages/sui-studio/bin/sui-studio-migrate.js

```diff
@@ -23,7 +23,8 @@ if (!topic) {
   console.log(
     'An argument is required. Please, try using "test" or "demo" topic'
   )
-  return
+
+  process.exit(1)
```

Comment:
For fixing with new `sui-lint` version (_remove parsing error return outside a function_)